test/integration/Dockerfile

```
@@ -1,11 +1,9 @@
 # This is not fully compatible builder Dockerfile, but meets the needs of tests
-FROM quay.io/centos/centos:stream9
-ARG WHEEL
-COPY $WHEEL /$WHEEL
+FROM quay.io/centos/centos:stream10
 RUN dnf install -y python3-pip
-RUN python3 -m pip install /$WHEEL ansible-core
+RUN python3 -m pip install ansible-core
```

Don't we need to have runner in the env still?
```suggestion
RUN python3 -m pip install . ansible-core
```

---

Ah, never mind. I just saw that it's not needed in the PR description.